cmu-sei/juneberry
# JB-372 extend DT2 config path

Added ability to specify full model config via module.
Added ability to add supplemental configs.
Updated docs.

To test:
- Run a dry run and save off the platform config.
- Copy config_with_sample_platform.json to config.json
- Run again and compare two platform configs.  The real difference should be the seed.
